script/test_k8s/kubernetes.sh

```diff
@@ -104,6 +104,13 @@ test_k8s() {
     echo -e "\n${RED}kompose down -f $f ${NC}\n"
     ./kompose down -f $f
   done
+
+  echo -e "\nTesting stdin to kompose\n"
```

Comment:
We shouldn't be adding these tests to integration tests... But more within CMD line tests.

Reply:
Argh. I saw what we did within #868 , see my comment there too. I'm fine with these tests (for now) but it's a bit of a hack / needs to refactor. But let's put that on our future agenda.

Reply:
@cdrage sure, I tried adding CMD line tests for stdin but got no success, so, I added here